Data/RAList.hs

```diff
@@ -1,3 +1,4 @@
+{-#LANGUAGE Safe #-}
```

Comment:
In this case safe might be ok. But for library code often simpler to mark as trustworthy